### PR TITLE
[10.x.x] Fixed XR mirror sRGB issue when colorSpace is gamma

### PR DESCRIPTION
**Pull request by thomas-zeng**
### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [ ] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [ ]  Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
- PR fixed sRGB in XR mirrorview when color space is gamma

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
- Adjust mirrorview logic to handle both sRGB read and sRGB write. Also added more comment to the code to make it easier to understand.
- To QA: I have tested Quest link on PC DX11 and verified mirrorview sRGB looks right

### CODE REVIEW

File: com.unity.render-pipelines.universal/Shaders/XR/XRMirrorView.hlsl [hunk 1]
```diff
@@ -43,8 +44,14 @@ float4 FragBilinear(Varyings input) : SV_Target
     outColor = SAMPLE_TEXTURE2D(_SourceTex, sampler_LinearClamp, input.texcoord.xy);
 #endif
 
+    if (_SRGBRead && _SRGBWrite)
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Shaders/XR/XRMirrorView.hlsl:47]: iirc uint and bitfield op don't work on gles2 but I guess that might not be a problem for you. Just make sure this shader doesn't compile for gles2.

Comment (thomas-zeng) [reply com.unity.render-pipelines.universal/Shaders/XR/XRMirrorView.hlsl:47]: I can confirm this shader does not compile for gles2: https://github.com/Unity-Technologies/Graphics/blob/12828751e058d5d82b944f387ea540d4ca02cdef/com.unity.render-pipelines.universal/Shaders/XR/XRMirrorView.shader#L8

**Review by DennisDeRykeUnity — APPROVED** [review]
I consider Thomas' testing (Quest in Link mode) to be sufficient given the problem at hand (mirror view totally broken), provided that automation becomes green (or as green as master).  I went ahead and launched a fresh SRP ABV here: 
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics/tree/universal%252Fxr%252Ffix-mirror-view-srgb/.yamato%252F_abv.yml%2523all_project_ci_fast-trunk/3466485/job 

@thomas-zeng Also note the automated comment in this PR regarding Conflicting files.

**Comment by thomas-zeng** [comment]
Yamato is green: https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/universal%252Fxr%252Ffix-mirror-view-srgb/.yamato%252F_abv.yml%2523all_project_ci_trunk/3508771/job
PR is ready to merge.

**Pull request merged**